src/userModifySrc.js

```diff
@@ -0,0 +1,125 @@
+const bcrypt = require('bcrypt');
+const db = require('../db/db');
+
+/**
+
+Update users
+@param {string} title Updating user
+@param {string} description Updating User information
```

Comment:
You don't have a description and title you have a name, email, organization, so change that.